### ISSUE TITLE
MacOs v0.12.0 double clicking definition fails in wiktionary

### ISSUE THREAD

**Issue by s-cork** [FreeLanguageTools/vocabsieve#162]
**Describe the bug**
Double click a word leads to no defintions

**To Reproduce**
1. Working on Mac 14.4.1
2. follow the installation steps for arm
3. (my target language is russian)
4. double click on a word
5. in 0.11.1 the wiktionary definition succeeds in 0.12.0 the wiktionary always fails

**Screenshots**
<img width="975" alt="Screenshot 2024-07-01 at 14 56 30" src="https://github.com/FreeLanguageTools/vocabsieve/assets/36813890/3ad9444b-4e80-446e-b208-96b662ed15d9">

**Logs**
Go to Help->View session logs, paste the log here. **Be sure to redact out any sensitive information. The log can contain clipboard content.**

VocabSieve version: 0.12.0
Python version: 3.11.7 (v3.11.7:fa7a6f2303, Dec  4 2023, 15:22:56) [Clang 13.0.0 (clang-1300.0.29.30)] on Darwin 23.4.0 arm64
PyQt5 (Qt bindings) version: 5.15.10, Qt 5.15.11

```
2024-07-01 14:58:22.938 | DEBUG    | vocabsieve.main:initSources:148 - Initializing sources
2024-07-01 14:58:22.939 | DEBUG    | vocabsieve.main:initSources:154 - Source Group 1: ['Wiktionary (English)'] has been created.
2024-07-01 14:58:22.940 | DEBUG    | vocabsieve.main:initSources:161 - Source Group 2: ['Google Translate'] has been created.
2024-07-01 14:58:22.942 | DEBUG    | vocabsieve.main:initSources:173 - Audio source group: ['Forvo'] has been created
2024-07-01 14:58:22.942 | DEBUG    | vocabsieve.main:initTimers:1049 - Initializing timers
2024-07-01 14:58:22.957 | DEBUG    | vocabsieve.main:getKnownDataOnThread:426 - Some data sources aren't available, not getting known data now
2024-07-01 14:58:22.957 | INFO     | vocabsieve.main:setupClipboardMonitor:110 - Clipboard monitoring is not supported on Wayland and MacOS, will poll instead
2024-07-01 14:58:36.651 | DEBUG    | vocabsieve.main:pollClipboard:137 - Polling: Clipboard text changed to '''Я хочу пройтись по магазинам'''
2024-07-01 14:58:40.401 | DEBUG    | vocabsieve.ui.searchable_boldable_text_edit:bold:11 - bolding хочу
2024-07-01 14:58:40.406 | DEBUG    | vocabsieve.ui.multi_definition_widget:lookup:135 - Looking up хочу in [<vocabsieve.sources.wiktionary_source.WiktionarySource object at 0x130980250>]
2024-07-01 14:58:40.406 | DEBUG    | vocabsieve.ui.multi_definition_widget:lookup:135 - Looking up хочу in [<vocabsieve.sources.google_translate_source.GoogleTranslateSource object at 0x130982e90>]
2024-07-01 14:58:40.407 | INFO     | vocabsieve.sources.wiktionary_source:_lookup:20 - Looking up хотеть in Wiktionary
2024-07-01 14:58:40.408 | INFO     | vocabsieve.sources.forvo_audio_source:_lookup:173 - Forvo lookup хотеть
2024-07-01 14:58:40.524 | INFO     | vocabsieve.sources.forvo_audio_source:_lookup:173 - Forvo lookup хочу
2024-07-01 14:58:41.047 | ERROR    | vocabsieve.sources.wiktionary_source:_lookup:26 - Failed to get data from Wiktionary: HTTPError('404 Client Error: Not Found for url: https://kaikki.org/dictionary/Russian/meaning/%D1%85/%D1%85%D0%BE/%D1%85%D0%BE%D1%82%D0%B5%D1%82%D1%8C.json')
2024-07-01 14:58:41.057 | INFO     | vocabsieve.sources.wiktionary_source:_lookup:20 - Looking up хочу in Wiktionary
2024-07-01 14:58:41.188 | DEBUG    | vocabsieve.ui.multi_definition_widget:run:70 - LookupWorker: looked up хочу in Google Translate in 0.78 seconds
2024-07-01 14:58:41.188 | DEBUG    | vocabsieve.ui.multi_definition_widget:appendDefinition:164 - All sources have been looked up
2024-07-01 14:58:41.705 | ERROR    | vocabsieve.sources.wiktionary_source:_lookup:26 - Failed to get data from Wiktionary: HTTPError('404 Client Error: Not Found for url: https://kaikki.org/dictionary/Russian/meaning/%D1%85/%D1%85%D0%BE/%D1%85%D0%BE%D1%87%D1%83.json')
2024-07-01 14:58:41.707 | DEBUG    | vocabsieve.ui.multi_definition_widget:run:70 - LookupWorker: looked up хочу in Wiktionary (English) in 1.30 seconds
2024-07-01 14:58:41.707 | DEBUG    | vocabsieve.ui.multi_definition_widget:appendDefinition:164 - All sources have been looked up
```

**Desktop (please complete the following information):**
 - OS: Mac 14.4.1
- Vocabsieve version (if nightly, must be latest): 0.12.0

**Comment by sindikat** [comment]
Links changed from `.json` to `.jsonl` as mentioned here: https://github.com/tatuylonen/wiktextract

See also #164

**Comment by parkerlouis** [comment]
Could someone please advise how to apply the fix in VocabSieve? I attempted to modify the `wiktionary_source.pyc` file by changing `.json` to `.jsonl` but this prevents the application from running. Any help you can provide is very much appreciated. Thank you in advance and thanks for creating this awesome program!

**Comment by 1over137** [comment]
The online lookup has been fixed for a while now. The import is fixed in 0.12.3

**Comment by parkerlouis** [comment]
@1over137 thanks for replying. I’ll try the new version. 

> The online lookup has been fixed for a while now. The import is fixed in 0.12.3

Thanks for replying. I’ll try the new version.
